src/chrome/content/rules/JapanTimes.co.jp.xml

```diff
@@ -0,0 +1,58 @@
+<!--
+	Non-functional hosts
+		Couldn't resolve host name:
+			 - asahi-ftp.japantimes.co.jp
+
+		Couldn't connect to server:
+			 - helpdesk.japantimes.co.jp
+			 - mw1.japantimes.co.jp
+			 - mw2.japantimes.co.jp
+			 - redmine.japantimes.co.jp
+			 - rt.japantimes.co.jp
+			 - svn.japantimes.co.jp
+
+		Timeout was reached:
+			 - blog.japantimes.co.jp
+			 - bugs.japantimes.co.jp
+			 - cms.japantimes.co.jp
+			 - features.japantimes.co.jp
+			 - ipm-archives.japantimes.co.jp
+			 - showcase.japantimes.co.jp
+			 - aomori.showcase.japantimes.co.jp
+			 - great-tamba.showcase.japantimes.co.jp
+			 - women.japantimes.co.jp
+
+		SSL peer certificate or SSH remote key was not OK:
+			 - www.japantimes.co.jp
+			 - realestate.japantimes.co.jp
+			 - support.japantimes.co.jp
+
+		Peer certificate cannot be authenticated with given CA certificates:
+			 - bookclub.japantimes.co.jp
+			 - genki.japantimes.co.jp
+			 - ij.japantimes.co.jp
```

Comment:
Fixed. 

For research purpose, why a `Incomplete certificate chain` has to be treated differently? When I use `curl` to test against the hosts, it throws an error before the redirections. However, Firefox does not. Is this a reason for that?